packages/docs/celo-holder-guide/alfajores-recovery.md

```diff
@@ -0,0 +1,73 @@
+# Recovering CELO from Alfajores Wallet
+
+In this guide, steps are outlined to help you recover your Mainnet [CELO (previously Celo Gold)](../overview.md#background-and-key-concepts) if you accidentally transferred them to your [Alfajores Testnet](../getting-started/alfajores-testnet.md) wallet.
+
+## Prerequisites
+
+This guide assumes that you have access to the following:
+
+- A computer with a [Command Line Interface](https://en.wikipedia.org/wiki/Command-line_interface). You can access it following these instructions:
+    - [Windows](https://www.howtogeek.com/270810/how-to-quickly-launch-a-bash-shell-from-windows-10s-file-explorer/)
+    - [Mac](https://macpaw.com/how-to/use-terminal-on-mac)
+    - [Linux](https://askubuntu.com/questions/196212/how-do-you-open-a-command-line)
+- [Celo Command Line Interface](../command-line-interface/introduction.md) installed on your computer
+- The [24-word recovery phrase](https://kb.myetherwallet.com/en/security-and-privacy/what-is-a-mnemonic-phrase/) (mnemonic) of your Alfajores wallet
+
+{% hint style="danger" %}
+There are [risks](https://www.cryptomathic.com/news-events/blog/cryptographic-key-management-the-risks-and-mitigations) associated with using a recovery phrase or a private key. Please do not share them with anyone else.
+{% endhint %}
+
+## Steps
+
+Please follow the instructions below closely, because missteps can lead to errors or permanent loss of your tokens.
+
+
+### Prepare your recovery phases
+
+Open the command-line interface on your computer. Write your recovery phrase to a file using the following commands:
+
+1. `nano recovery.txt`
+2. Paste `<word1> <word2> … <word24>`
+3. Replace the `<word>`s in brackets with the 24 words from your recovery phrase (mnemonic)
+4. Press ctrl-o to save
+5. Press ctrl-x to exit
+
+### Recover your Alfajores address on Celo
+
+Recover your Alfajores address on Celo Mainnet:
+
+```
+celocli account:recover-old --mnemonicPath recovery.txt --node https://rc1-forno.celo-testnet.org
```

Comment:
Might be worth chatting with Ponti about making this be a flag passed to account:new? e.g. --legacyWallet?

Reply:
will chat with Ponti, but I doubt the change will be rolled out anytime soon.

Reply:
@marekolszewski @gastonponti Is it ok to start pointing users to this document?

Reply:
yes